roles/postgres/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+- name: Install postgres packages.
+  apt:
+    name:
+      - python3-psycopg2
+      - postgresql-{{ postgresql_version }}
+      - postgresql-contrib-{{ postgresql_version }}
+      - libpq-dev
+    state: present
+
+- name: Check postgres is started and enabled on boot.
+  service:
+    name: '{{ postgresql_daemon }}'
+    state: started
+    enabled: true
+
+- name: Add postgres users.
+  community.postgresql.postgresql_user:
+    name: "{{ item.name }}"
```

Comment:
Instead of manually mapping each value, you can expand the entire dict. See my ufw PR for reference.

Reply:
Oh, this is very nice, thanks! Should help us if we want to add more in future, since the built-in supports quite a few fields